src/gpu/intel/jit/gemm/gen_gemm.hpp

```diff
@@ -144,7 +144,7 @@ struct gen_gemm_t : public gpu_gemm_t {
             int b_ndims = desc()->b_desc.ndims;
 
             // Check parameters.
-            if (utils::one_of(d->c_type(), s32, f16, f32, u8, s8)
+            if (utils::one_of(d->c_type(), s32, f16, bf16, f32, u8, s8)
```

Comment:
Should this be need enabled in `ref_gemm` as well?

Reply:
It probably could be, but since ref_matmul/ref_inner_product already cover it, I don't think we need to worry about it.